src/App.tsx

```diff
@@ -1,7 +1,21 @@
-import { greet } from "./utils/greet";
```

Comment:
Nice job removing redundant code here

Reply:
Thank you, it was a difficult decision, but ultimately the right move on balance, we think.